associated-token-account/program/src/instruction.rs

```diff
@@ -0,0 +1,51 @@
+//! Program instructions
+
+use borsh::{BorshDeserialize, BorshSchema, BorshSerialize};
+use solana_program::{
+    instruction::{AccountMeta, Instruction},
+    pubkey::Pubkey,
+    sysvar,
+};
+
+use crate::{get_associated_token_address, id};
+
+/// Instructions supported by the AssociatedTokenAccount program
+#[derive(Clone, Debug, PartialEq, BorshDeserialize, BorshSerialize, BorshSchema)]
+pub enum AssociatedTokenAccountInstruction {
+    /// Creates an associated token account for the given wallet address and token mint
+    ///
+    ///   0. `[writeable,signer]` Funding account (must be a system account)
+    ///   1. `[writeable]` Associated token account address to be created
+    ///   2. `[]` Wallet address for the new associated token account
+    ///   3. `[]` The token mint for the new associated token account
+    ///   4. `[]` System program
+    ///   5. `[]` SPL Token program
+    ///   6. `[]` Rent sysvar
+    CreateAssociatedTokenAccount,
```

Comment:
```suggestion
    Create,
```
(no need to repeat the "AssociatedTokenAccount" in the enum name)

Reply:
Done